packages/discovery-react-components/src/components/SearchResults/utils/__tests__/getDocumentTitle.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { getDocumentTitle } from '../index';
+
+const mockResultWithTitle = {
```

Comment:
In order to increase readability, we could move these constants to sit right above their tests.

Reply:
sure thing 👍